packages/web/src/pages/sign-on/components/desktop/LeftContentContainer.module.css

```diff
@@ -0,0 +1,4 @@
+.root {
+  background-color: var(--harmony-white);
```

Comment:
Heads up we can use emotion now for some of this

Reply:
this is in web, we dont have emotion there right?

Reply:
we do now! we can chat on monday on what that could look like